src/vsl/parser/grammar/ws.ne

```diff
@@ -1 +1,2 @@
-_ -> "\n":*
+_
+   -> "\n":*
```

Comment:
Does this really need to be on another line? Either way, does this have a trailing newline at the end of file, I can't tell if it's just github being weird from online.